Essentials/src/com/earth2me/essentials/Essentials.java

```diff
@@ -767,6 +772,9 @@ private int broadcastMessage(final IUser sender, final String permission, final
         for (Player player : players) {
             final User user = getUser(player);
             if ((permission == null && (sender == null || !user.isIgnoredPlayer(sender))) || (permission != null && user.isAuthorized(permission))) {
+                if (exclude.length > 0 && Arrays.asList(exclude).contains(user)) {
```

Comment:
For this particular use case, it's not a terribly big deal, but first of all, don't create a new `ArrayList` every time you get to a new player. Second, perhaps consider using something like a `HashSet` in the off chance that someone uses this method with a large `exclude` array.